contracts/PlotCustodianManager.sol

```diff
@@ -113,6 +113,10 @@ contract PlotCustodianManager is AbstractApplication {
     _;
   }
 
+  function setGasPriceForDeposits(uint256 _newPrice) external onlyFeeManager {
+    gasPriceForDeposits = _newPrice;
+  }
+
```

Comment:
We don't need deposit functionality in this contract

Reply:
ok